server/CGameHandler.cpp

```diff
@@ -669,6 +669,19 @@ void CGameHandler::onPlayerTurnEnded(PlayerColor which)
 		heroPool->onNewWeek(which);
 }
 
+void CGameHandler::addStatistics()
+{
+	for (auto & elem : gs->players)
+	{
+		if (elem.first == PlayerColor::NEUTRAL || !elem.first.isValidPlayer())
+			continue;
```

Comment:
I think neither case is possible here

Reply:
In the parent function both there are checked on other place: https://github.com/vcmi/vcmi/blob/c8b48318b7a511c5a9a26812d4b9c26494a18f9b/server/CGameHandler.cpp#L759